packages/ui/src/theme/css/component/passwordField.scss

```diff
@@ -1,44 +1,51 @@
-.amplify-passwordfield {
```

Comment:
Do we know why this changed? It appears to be identical. Was it just formatting?

Reply:
it formatting only i guess